sonarpedia.json

```diff
@@ -1,11 +1,11 @@
 {
   "rules-metadata-path": "./sonar-html-plugin/src/main/resources/org/sonar/l10n/web/rules/Web",
   "languages": [
-    "WEB"
```

Comment:
I think we should keep WEB, it's working fine and we avoid unexpected side-effects. In addition, we should not bother too much with rule-api magic since it will (hopefully) be replaced soon.